CHANGELOG.md

```diff
@@ -3,14 +3,10 @@ We follow the [Semantic Versioning 2.0.0](http://semver.org/) format.
 
 ## Unreleased
 
-### Added
-- 
-
 ### Changed
-- 
+- **cf-notifications:** Renamed from `cf-notification`
+  to `cf-notifications`.
```

Comment:
We didn't add anything to the changelog about updating the pull request template buuuuut I'm okay with that because I'm lazy and it's a meta change that doesn't affect the framework's code.

Reply:
Yeah not a change that's part of the downloadable modules.